app/src/main/java/com/teamwss/websoso/ui/onboarding/OnboardingViewModel.kt

```diff
@@ -86,12 +92,34 @@ class OnboardingViewModel @Inject constructor(
     }
 
     private fun updateUIByPage(page: OnboardingPage) {
-        _progressBarPercent.value = page.progressPercent
-        _isBackButtonVisible.value = page.isBackButtonVisible
-        _isSkipTextVisible.value = page.isSkipTextVisible
+        with(page) {
+            _progressBarPercent.value = progressPercent
+            _isBackButtonVisible.value = isBackButtonVisible
+            _isSkipTextVisible.value = isSkipTextVisible
+        }
     }
 
     fun updateUserBirthYear(birthYear: Int) {
-        _userBirthYear.value = birthYear
+        _userInfo.value = _userInfo.value?.copy(birth = birthYear)
```

Comment:
```suggestion
        _userInfo.value = userInfo.value?.copy(birth = birthYear)
```
c: 이런식으로 사용할 때는 언더바를 빼주는게 맞다네요...!

Reply:
c: 그리구 birth = birthYear이란 말이 어딘가 어색해요!!

Reply:
서버에서 birth만 사용해서 그대로 차용하였는데 UiModel은 이름을 변경하는 것을 고려해보겠습니다.